src/olympia/abuse/models.py

```diff
@@ -44,6 +45,10 @@ class CinderJob(ModelBase):
         ('AMO_DELETE_COLLECTION', 6, 'Collection delete'),
         ('AMO_APPROVE', 7, 'Approved (no action)'),
     )
+    DECISION_ACTIONS.add_subset(
+        'UNRESOLVED',
+        ('NO_DECISION', 'AMO_ESCALATE_ADDON'),
```

Comment:
That looks weird. Is that what you meant ?

Reply:
It was - does it not make sense? These are reports that need a decision still.

Reply:
You're right, misread that